core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -26,132 +26,180 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form" xmlns:l="/lib/layout" xmlns:i="jelly:fmt" xmlns:d="jelly:define">
     <d:taglib uri="local">
-        <d:tag name="revokeIcon">
-          <l:icon src="symbol-trash" />
+        <d:tag name="editButton">
+            <button type="button"
+                    class="jenkins-table-button jenkins-button jenkins-button--tertiary api-token-property-token-rename"
+                    data-target-url="${descriptor.descriptorFullUrl}/rename" data-rename-title="${%RenameToken}"
+                    data-rename-message="${%New name}"
+                    tooltip="${%Rename this token}">
+                <l:icon src="symbol-edit"/>
+            </button>
+        </d:tag>
+        <d:tag name="revokeButton">
+            <button type="button"
+                    class="jenkins-table-button jenkins-button jenkins-button--tertiary jenkins-!-destructive-color api-token-property-token-revoke"
+                    data-message-if-legacy-revoked="${%RevokedToken}"
+                    data-confirm="${%ConfirmRevokeSingle}"
+                    data-confirm-title="${%ConfirmRevokeSingleTitle}"
+                    data-token-uuid="${token.uuid}"
+                    data-target-url="${descriptor.descriptorFullUrl}/revoke" tooltip="${%RevokeToken}">
+                <l:icon src="symbol-trash" />
+            </button>
         </d:tag>
     </d:taglib>
+    <template id="api-token-row-template" xmlns:local="local">
+        <tr>
+            <td>
+                <div class="token-name-container">
+                    <span class="token-name"/>
+                </div>
+            </td>
+            <td>
+                <span class="token-creation-date" tooltip="${%Just now}">${%Today}</span>
+            </td>
+            <td align="right" class="token-use-counter">0</td>
+            <td align="right">
+                <div class="token-last-used">${%Never}</div>
+            </td>
+            <td class="jenkins-table__cell--tight">
+                <div class="jenkins-table__cell__button-wrapper">
+                    <local:editButton/>
+                </div>
+            </td>
+            <td class="jenkins-table__cell--tight">
+                <div class="jenkins-table__cell__button-wrapper">
+                    <local:revokeButton/>
+                </div>
+            </td>
+        </tr>
+    </template>
+    <div class="jenkins-hidden" id="api-token-template">
+        <div class="api-token-form">
+            ${%Name}: <span class="api-token-new-name"/>
+            <div class="api-token-new-form-row">
+                <span>${%Token}:</span>
```

Comment:
I think this would look better without the `Name:` and token prefix.

Move the name into the heading and drop the Token, 

e.g. 'my-token successfully created' for the heading

<img width="684" alt="image" src="https://github.com/user-attachments/assets/b9755da5-9816-4edf-871b-9dbe7aa28200" />

Reply:
Putting the name in the title is a bit tricky from translation perspective I think. Would need to be handled properly also for right to left languages, and not sure if in any of the languages it would look better if the token name is not at the beginning. 
Of course only if someone will ever translate those things then

Reply:
I don't think thats stopped us in other places before, at least when its at the end:

<img width="979" alt="image" src="https://github.com/user-attachments/assets/883a1ad6-448e-48ba-badc-2a189c80f799" />

Could just use the token name too without the next?

(assuming we keep this dialog)